sysdig/data_source_sysdig_fargate_workload_agent.go

```diff
@@ -0,0 +1,208 @@
+package sysdig
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/falcosecurity/kilt/runtimes/cloudformation/cfnpatcher"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const agentinoKiltDefinition = `build {
+    entry_point: ["/opt/draios/bin/instrument"]
+    command: ${?original.entry_point} ${?original.command}
+    environment_variables: {
+        "SYSDIG_ORCHESTRATOR": ${config.orchestrator_host}
+        "SYSDIG_ORCHESTRATOR_PORT": ${config.orchestrator_port}
+        "SYSDIG_COLLECTOR": ${config.collector_host}
+        "SYSDIG_COLLECTOR_PORT": ${config.collector_port}
+        "SYSDIG_ACCESS_KEY": ${config.sysdig_access_key}
```

Comment:
Eric Lugo and I briefly looked at this. Does it make sense to move these env variables to be part of the top level provider config where we currently store the api tokens for monitor and secure?

Reply:
No idea. Right now if you want to provision multiple agentinos with different config, you get that for free (as you have to put these params in the data source). If we move these to the provider config, we'll have to set up multiple providers if we want different agentino configs. I'm not sure if terraform modules can easily create new provider instances (they probably can but there was something not 100% obvious about it)

Reply:
these settings are per VPC we want to monitor, so they need to be more specific

Reply:
What about the Access Collector and Access Key? Those will not change

Reply:
I'm not convinced splitting the settings across the provider/data source will be more intuitive but if you feel strongly about it, just tell me the list of settings you want moved to the provider